python-package/README.rst

```diff
@@ -110,6 +110,17 @@ All available options:
 
 For more details see `FindBoost <https://cmake.org/cmake/help/latest/module/FindBoost.html>`__ and `FindOpenCL <https://cmake.org/cmake/help/latest/module/FindOpenCL.html>`__.
 
+Build CUDA Version
+~~~~~~~~~~~~~~~~~~
+
+.. code:: sh
+
+    pip install lightgbm --install-option=--cuda
+
+All remarks from `Build from Sources section <#build-from-sources>`__ are actual in this case and `CMake`_ (version 3.16 or higher) is strongly required.
```

Comment:
```suggestion
All requirements from `Build from Sources section <#build-from-sources>`__ apply for this installation option as well, and `CMake`_ (version 3.16 or higher) is strongly required.
```

Reply:
This is standard phrase across this doc. Again, if you do not mind I'll change it in a separate PR to keep the doc consistent.

Reply:
ah ok no problem. I was just taking the diff as-is...didn't realize some other these things were patterns reused elsewhere in the doc. Changing them in a different PR is ok and not urgent.